migrations/1664278604452_add-table-to-record-flagged-posts.js

```diff
@@ -0,0 +1,27 @@
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+	pgm.createTable('post_flags', {
+		userId: {
+			type: 'integer',
+			notNull: true,
+			references: '"users"',
+			onDelete: 'cascade'
+		},
+		postId: {
+			type: 'integer',
+			notNull: true,
+			references: '"posts"',
```

Comment:
now seeing this, the double quotes are not too necessary here.

Reply:
okay, i will make the change as required